src/AspNet.Security.OAuth.Harvest/HarvestAuthenticationExtensions.cs

```diff
@@ -0,0 +1,76 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using AspNet.Security.OAuth.Harvest;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Authentication;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods to add GitHub authentication capabilities to an HTTP application pipeline.
+    /// </summary>
+    public static class GitHubAuthenticationExtensions
```

Comment:
`GitHub` needs replacing with `Harvest` in various code and comment places in this file.